engine/src/main/java/org/terasology/logic/players/CameraClientSystem.java

```diff
@@ -65,15 +98,38 @@ public void resetCamera() {
     }
 
```

Comment:
The default is preferably specified in the camera.prefab.  Instead of grabbing the prefab definition, and then cloning a new component on to the existing entity.  I thought it would be cleaner to just wipe it and start over so that there is only one code path possibility for how the location default is retrieved instead of two.